pkg/cmd/repo/clone/clone_test.go

```diff
@@ -0,0 +1,195 @@
+package clone
+
+import (
+	"bytes"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/cli/cli/internal/config"
+	"github.com/cli/cli/pkg/cmdutil"
+	"github.com/cli/cli/pkg/httpmock"
+	"github.com/cli/cli/pkg/iostreams"
+	"github.com/cli/cli/test"
+	"github.com/google/shlex"
+	"github.com/stretchr/testify/assert"
+)
+
+// TODO copypasta from command package
+type cmdOut struct {
+	outBuf, errBuf *bytes.Buffer
+}
+
+func (c cmdOut) String() string {
+	return c.outBuf.String()
+}
+
+func (c cmdOut) Stderr() string {
+	return c.errBuf.String()
+}
+
+func runCloneCommand(httpClient *http.Client, cli string) (*cmdOut, error) {
```

Comment:
My thinking is that each new isolated command will come with a `run*Command` function that replaces use of `runCommand` in tests. This worked well here.

Reply:
Great approach! I would argue that we could even name it `RunCommand` scoped to the new package so we don't have to rename any existing `RunCommand` calls from tests that we copy over 🤘